db/migrate/20230215074423_move_user_settings.rb

```diff
@@ -0,0 +1,74 @@
+class MoveUserSettings < ActiveRecord::Migration[6.1]
+  class User < ActiveRecord::Base; end
+
+  MAPPING = {
+    default_sensitive: 'web.default_sensitive',
```

Comment:
Several settings are missing and thus won't be migrated:

```suggestion
  MAPPING = {
    default_privacy: 'default_privacy',
    default_language: 'default_language',
    noindex: 'noindex',
    theme: 'theme',
    trends: 'web.trends',
    default_sensitive: 'web.default_sensitive',
```